webapp/pages/settings/my-social-media.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <ds-card :header="$t('settings.social-media.name')">
+    <ds-space
+      v-if="socialMediaLinks"
+      margin-top="base"
+      margin="x-small"
+    >
+      <ds-list>
+        <ds-list-item
+          v-for="link in socialMediaLinks"
+          :key="link.url"
+        >
+          <a :href="link.url">
+            <img
+              :src="link.favicon"
+              alt=""
+            >
+            {{ link.url }}
```

Comment:
should this be the entire url @roschaefer, we were thinking of either just the username, or just having the icon as the link
![Screenshot at 2019-03-28 14:43:48](https://user-images.githubusercontent.com/26943915/55180228-ec553800-5167-11e9-838f-3bb75ce53c6a.png)

Reply:
I think it has some advantage to see the url, people like to know where the icon points to.

If you know how to get the username out of an arbitrary url, go for it :thinking: :woman_shrugging:

Reply:
I prefer that only the username is filtered from the URL and displayed with the icon.
Otherwise it might get too long.

Reply:
@mattwr18 if it turns out that filtering the username from the URL is not that easy, don't spend too much time on it. I agree with @Lulalaby but a quick fix could also be that you simply shorten the URL like `/some/path...`.

Reply:
I think potential it could be not too complicated @roschaefer... the regex that @Kachulio1 wrote :point_up: separates the the beginning of the URL from the username already, so potentially we just need to save what it cuts as the username, but we'll have a look at it

Reply:
yeah, just do it! 😸